src/DynamoSandbox/app.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+﻿<?xml version="1.0"?>
```

Comment:
Please do a `git checkout master src/DynamoSandbox/app.config` to revert this file, it is unnecessarily polluting the PR.